### PR TITLE
Fixed bug, that occured when starting one of the detectors VTD, VCD a…

### LINKED ISSUE #918: ETD: Make following detectors independent from ETD persistency

### PR DESCRIPTION
**Pull request by hoeldg**
…nd TSA with an already existing persistency of the ETD, but not of the detectors.

# Make sure these boxes are signed before submitting your Pull Request -- thank you.

# Must haves
- [x] I have read and followed the contributing guide lines at https://github.com/ait-aecid/logdata-anomaly-miner/wiki/Git-development-workflow
- [x] Issues exist for this PR
- [x] I added related issues using the "Fixes #<issue-id>"-notations
- [x] This Pull-Requests merges into the "development"-branch

Fixes #918 

# Submission specific

- [ ] This PR introduces breaking changes
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

# Describe changes:
-

### CODE REVIEW

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/TSAArimaDetector.py [hunk 1]
```diff
@@ -225,21 +223,21 @@ def receive_atom(self, log_atom):  # skipcq: PYL-W0613, PYL-R0201
...
-                            for j, val in enumerate(self.event_type_detector.num_eventlines):
-                                self.num_eventlines_ref[j].append(val-sum(self.num_eventlines_ref[j]))
+                        # Expand the lists of self.num_event_lines_ref
+                        for j in range(len(self.num_event_lines_ref), len(self.event_type_detector.num_event_lines)):  # skipcq: PTC-W0060
+                            self.num_event_lines_ref.append([0]*len(self.num_event_lines_ref[0]))
+                        # Add the current number of eventlines
+                        for j, val in enumerate(self.event_type_detector.num_event_lines):
+                            self.num_event_lines_ref[j].append(val-sum(self.num_event_lines_ref[j]))
+
+                        print(self.num_event_lines_ref)
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/TSAArimaDetector.py:233]: Remove prints

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/VariableCorrelationDetector.py [hunk 2]
```diff
@@ -490,6 +498,7 @@ def init_cor_rel(self, event_index):
                 if (j_val in self.rel_list[event_index][pos_var_cor_index][1]) and (i_val in self.rel_list[event_index][
                         pos_var_cor_index][1][j_val]):
                     self.rel_list[event_index][pos_var_cor_index][1][j_val][i_val] += 1
+        print(self.rel_list[event_index])
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/VariableCorrelationDetector.py:501]: Remove print

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/TSAArimaDetector.py (hunk 1 above)
Comment (hoeldg) [reply source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/TSAArimaDetector.py]: Removed prints

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/VariableCorrelationDetector.py (hunk 2 above)
Comment (hoeldg) [reply source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/VariableCorrelationDetector.py]: Removed prints

**Review by landauermax — APPROVED** [review]
Looks good to me

**Pull request merged**